PWGHF/HFL/Tasks/taskHFmuCharmBeautySep.cxx

```diff
@@ -0,0 +1,97 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+///
+/// \file taskHFmuCharmBeautySep.cxx. This workflow requires o2-analysis-fwdtrackextension as a dependency.
+/// \brief Task to estimate HF->mu in the forward direction and use DCA observable to separate b-> mu, c-> mu.
+/// \author Shreyasi Acharya <shreyasi.acharya@cern.ch>, LPC, France
+
+#include "Framework/runDataProcessing.h"
+#include "Framework/AnalysisTask.h"
+#include "TDatabasePDG.h"
+#include "CCDB/BasicCCDBManager.h"
+#include "Framework/AnalysisDataModel.h"
+#include "ReconstructionDataFormats/TrackFwd.h"
+#include "Common/DataModel/TrackSelectionTables.h"
```

Comment:
Sort the includes

Reply:
See https://aliceo2group.github.io/analysis-framework/docs/advanced-specifics/pwghf.html#code-development-guidelines

Reply:
done